api/admin.go

```diff
@@ -112,7 +113,7 @@ func getAllAudits(c *Context, w http.ResponseWriter, r *http.Request) {
 		audits := result.Data.(model.Audits)
 		etag := audits.Etag()
 
-		if HandleEtag(etag, w, r) {
+		if HandleEtag(etag, "Get All Audits", w, r) {
```

Comment:
Yup. These are simply labels for the graphs. The Prometheus labels are separate. 
Tested and it doesn't break anything so all good!